connector-sdk/core/src/main/java/io/camunda/connector/api/annotation/OutboundConnector.java

```diff
@@ -30,7 +30,7 @@
   String name();
 
   /** Input variables the connector reads */
-  String[] inputVariables();
+  String[] inputVariables() default "";
```

Comment:
Should be `default {}` perhaps? II'm not sure but I suspect one resolves to `[ "" ]`, the other to `[]` 🤔